ci/deploy.yml

```diff
@@ -79,7 +79,9 @@ msg-system deploy dev:
   tags:
     - deploy
   script:
+    - source ${MSG_SYSTEM_DATA_DEPLOY_KEYFILE}
     - aws s3 cp packages/suite-data/files/message-system/config.v1.jws s3://data.trezor.io/config/develop/config.v1.jws
+    - aws cloudfront create-invalidation --distribution-id E1ERY5K2OTKKI1 --paths '/config/develop/*'
```

Comment:
Can we extrapolate the distribution id to a variable just for a better clarity?

Reply:
Yes, that's better. What about having the script which we would run in this job? Or it's unnecessary over-complication? 
ad variable: `AWS_CLOUDFRONT_DATA_DID` ?

Reply:
Maybe just AWS_CLOUDFRONT_ID?